charts/rancher-turtles/templates/core-provider-configmap.yaml

```diff
@@ -15142,7 +15142,7 @@ data:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.uid
-            image: registry.rancher.com/rancher/cluster-api-controller:v1.10.6
+            image: {{ template "system_default_registry" . }}rancher/cluster-api-controller:v1.10.6
```

Comment:
This might be overridden by the `clusterctlconfig` CR

Reply:
You are right, we need a different approach for this. I will rework this PR.